src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -28,103 +44,16 @@
-    </None>
-    <None Update="ReturnValueConfigurationExtensions.StronglyTyped.tt">
-      <Generator>TextTemplatingFileGenerator</Generator>
-      <LastGenOutput>ReturnValueConfigurationExtensions.StronglyTyped.cs</LastGenOutput>
-    </None>
-    <None Update="ValueTaskAsyncReturnValueConfigurationExtensions.StronglyTyped.tt">
-      <Generator>TextTemplatingFileGenerator</Generator>
-      <LastGenOutput>ValueTaskAsyncReturnValueConfigurationExtensions.StronglyTyped.cs</LastGenOutput>
-    </None>
-    <None Update="ValueTaskReturnValueConfigurationExtensions.StronglyTyped.tt">
-      <Generator>TextTemplatingFileGenerator</Generator>
-      <LastGenOutput>ValueTaskReturnValueConfigurationExtensions.StronglyTyped.cs</LastGenOutput>
-    </None>
-    <Compile Update="ArgumentValidationConfigurationExtensions.StronglyTyped.cs">
-      <DesignTime>True</DesignTime>
-      <AutoGen>True</AutoGen>
-      <DependentUpon>ArgumentValidationConfigurationExtensions.StronglyTyped.tt</DependentUpon>
-    </Compile>
-    <Compile Update="Configuration\ExceptionThrower.StronglyTyped.cs">
-      <DesignTime>True</DesignTime>
-      <AutoGen>True</AutoGen>
-      <DependentUpon>ExceptionThrower.StronglyTyped.tt</DependentUpon>
-    </Compile>
-    <Compile Update="Configuration\OutAndRefParameters.StronglyTyped.cs">
-      <DesignTime>True</DesignTime>
-      <AutoGen>True</AutoGen>
-      <DependentUpon>OutAndRefParameters.StronglyTyped.tt</DependentUpon>
-    </Compile>
-    <Compile Update="AsyncReturnValueConfigurationExtensions.StronglyTyped.cs">
-      <DesignTime>True</DesignTime>
-      <AutoGen>True</AutoGen>
-      <DependentUpon>AsyncReturnValueConfigurationExtensions.StronglyTyped.tt</DependentUpon>
-    </Compile>
-    <Compile Update="CallbackConfigurationExtensions.StronglyTyped.cs">
-      <DesignTime>True</DesignTime>
-      <AutoGen>True</AutoGen>
-      <DependentUpon>CallbackConfigurationExtensions.StronglyTyped.tt</DependentUpon>
-    </Compile>
-    <Compile Update="ReturnValueConfigurationExtensions.StronglyTyped.cs">
-      <DesignTime>True</DesignTime>
-      <AutoGen>True</AutoGen>
-      <DependentUpon>ReturnValueConfigurationExtensions.StronglyTyped.tt</DependentUpon>
-    </Compile>
-    <Compile Update="ValueTaskAsyncReturnValueConfigurationExtensions.StronglyTyped.cs">
-      <DesignTime>True</DesignTime>
-      <AutoGen>True</AutoGen>
-      <DependentUpon>ValueTaskAsyncReturnValueConfigurationExtensions.StronglyTyped.tt</DependentUpon>
-    </Compile>
-    <Compile Update="ValueTaskReturnValueConfigurationExtensions.StronglyTyped.cs">
-      <DesignTime>True</DesignTime>
-      <AutoGen>True</AutoGen>
-      <DependentUpon>ValueTaskReturnValueConfigurationExtensions.StronglyTyped.tt</DependentUpon>
-    </Compile>
-  </ItemGroup>
-
-  <ItemGroup Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'netstandard2.1'))">
```

Comment:
it became more convenient to deal with these files via ifdefs, since the generated code was incorporated whether we "Remove"d the files or not. An alternative would've been to generate different code for various TFMs, but that seemed annoying and confusing!